txn.go

```diff
@@ -336,30 +338,39 @@ func (txn *txn) delete(meta TableMeta, guardRevision Revision, data any) (object
 	return obj, true, nil
 }
 
+var primaryKeyEncodedLength = base64.StdEncoding.EncodedLen(32)
+
 // encodeNonUniqueKey constructs the internal key to use with non-unique indexes.
 // It concatenates the secondary key with the primary key and the length of the secondary key.
```

Comment:
needs comment on how the primary part is maybe hashed and base64 encoded

Reply:
Redid this using a different schema and documented.